airflow/models/dagbag.py

```diff
@@ -269,7 +270,12 @@ def _load_modules_from_file(self, filepath, safe_mode):
                 return [new_module]
             except Exception as e:  # pylint: disable=broad-except
                 self.log.exception("Failed to import: %s", filepath)
-                self.import_errors[filepath] = str(e)
+                if conf.getboolean('webserver', 'dagbag_import_error_tracebacks'):
```

Comment:
I put the config under webserver as I thought that made the most sense for the end user. Happy to move them elsewhere if needed.

Reply:
Perhaps just under `core`? (This feels slightly wrong here as this code isn't actually run in the webserver on 2.0, but under the scheduler)

Reply:
Yeah, looking with fresh eyes, `core` is a much better fit.

Think we should also set attributes like we do for `DAGBAG_IMPORT_TIMEOUT` so we only hit the config system once?
e.g. https://github.com/apache/airflow/blob/master/airflow/models/dagbag.py#L80

Reply:
@ashb, does 👍 mean you think we should do attrs?